src/microbit/__model/display.py

```diff
@@ -226,13 +227,19 @@ def is_on(self):
 
     def read_light_level(self):
         """
-        Not implemented yet.
-
         Use the display's LEDs in reverse-bias mode to sense the amount of light
         falling on the display.  Returns an integer between 0 and 255 representing
         the light level, with larger meaning more light.
         """
-        raise NotImplementedError(CONSTANTS.NOT_IMPLEMENTED_ERROR)
+        return self.__light_level
+
+    def __set_light_level(self, level):
+        if level < CONSTANTS.MIN_LIGHT_LEVEL:

```

Comment:
Is this how we should handle out of range values? Or should we throw an error

Reply:
Great question. Not sure to be honest. On the CPX, if I inputted a value less than the min (on the simulator), it defaulted to the min value, which is why I had it here.

Reply:
I see, maybe one of the coaches has advice on this. I am not sure either. I feel like it could "hide" some incorrect inputs if that ever happens

Reply:
I think that we want to do make sense of whatever the real thing does. It wouldn't read a value that is below it's minimum level -- so I'd suggest throwing an exception if that's the case.

Reply:
Okay, I will update it to raise an exception :)